lucene/analysis/nori/src/java/org/apache/lucene/analysis/ko/KoreanTokenizer.java

```diff
@@ -514,6 +522,9 @@ private void parse() throws IOException {
     // Index of the last character of unknown word:
     int unknownWordEndIndex = -1;
 
+    /// Largest global posAhead of user word
+    int global_posAheadMax = -1;
```

Comment:
Why is it needed ?

Reply:
Please refer to the below (1) comment.

Reply:
Got it, thanks for explaining. Can you rename with camel case for consistency: `userWordMaxPosAhead` ?

Reply:
Yeap!